src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
```

Comment:
Same as the question in `Duke.java`. Perhaps we can put classes in a package `task` etc.